extensions/azurePublishNew/src/node/availableResources.ts

```diff
@@ -27,6 +27,22 @@ export type AppRegistrationResourceConfig = ResourceConfig & {
   appName: string;
 };
 
+export type WebAppResourceConfig = ResourceConfig & {
```

Comment:
Consider putting these in their respective modules.

Reply:
fixed